### PR TITLE
Support speech recognition permission

### LINKED ISSUE #19: Support speech recognition permission
Could you add a permission of speech recognition? As I know, we can use `kTCCServiceSpeechRecognition` for speech recognition.

### PR DESCRIPTION
**Pull request by ghsdh3409**
### Issue
* It closes #19.

### Implementation
* Add speech recognition permission

### Test
* I checked that it works on the iOS 11 simulator.

### CODE REVIEW

**Comment by LeoNatan** [comment]
Hey,

Thanks for your PR.
I don’t think you made all the commits yet, as I don’t see anything changed.

File: applesimutils/applesimutils/main.m [hunk 1]
```diff
@@ -230,6 +230,7 @@ static void performPermissionsPass(NSString* permissionsArgument, NSString* simu
 																   @"motion": @"kTCCServiceMotion",
 																   @"health": @"kTCCServiceMSO",
 																   @"siri": @"kTCCServiceSiri",
+																   @"speech": @"kTCCServiceSpeechRecognition",
```
Comment (ghsdh3409) [inline applesimutils/applesimutils/main.m]: @LeoNatan As just adding `kTCCServiceSpeechRecognition`, we can use a speech recognition permission. The other logic is shared with other permission.

Comment (LeoNatan) [reply applesimutils/applesimutils/main.m]: Holy shit, what a bad UX GitHub has. I didn't see there is a scrollbar to see the actual text. 🤦‍♂️
Sorry.

File: README.md [hunk 2]
```diff
@@ -47,4 +47,5 @@ Available Permissions:
     photos=YES|NO|unset
     reminders=YES|NO|unset
     siri=YES|NO|unset
+    speech=YES|NO|unset
```
Comment (LeoNatan) [inline README.md]: Can you please add this to the usage builder in `main.m`?
Thanks

Comment (ghsdh3409) [reply README.md]: Sorry for late response. I just reflected this review.

**Comment by LeoNatan** [comment]
Thanks!

**Pull request merged**